docs/spark-queries.md

```diff
@@ -394,3 +394,22 @@ spark.read.format("iceberg").load("db.table.files").show(truncate = false)
 // Hadoop path table
 spark.read.format("iceberg").load("hdfs://nn:8020/path/to/table#files").show(truncate = false)
```

Comment:
Sorry its unrelated, but could you also remove the show (truncate=true) here?  Should have added the comment in the previous pr.  This doc should just explain how to load the dataframe, not necessarily how to show it.